### ISSUE TITLE
 MPEG_DASH manifest with audio track doesn't play in dash.js

### ISSUE THREAD

**Issue by alex-rsk** [gpac/gpac#2815]
- [++ ] I looked for a similar issue and couldn't find any.
- [ +] I tried with the latest version of GPAC. Installers available at https://gpac.io/downloads/gpac-nightly-builds/
- [ +] I give enough information for contributors to reproduce my issue (meaningful title, github labels, platform and compiler, command-line ...). I can share files anonymously 

Given:  video fragments of equal length, prepared from some source stream by segmenting into 10s chunks. 
Example command of preparation:
```
ffmpeg	-loglevel  verbose -i <SOURCE_URL>  -c:v  libx264  -c:a  aac -preset veryfast -force_key_frames 'expr:gte(t,n_forced*2)'
-f  segment	-segment_time 10  -reset_timestamps 1	-map 0  "/chunks/seg_%03d.mp4"
```
So we have seg_0001.mp4, seg_0002.mp4 etc.
You can download prepared files from here: [link](https://www.dropbox.com/scl/fo/024n472bairtmuz2e1lzi/AHn5Gj0sVwcTGfMn4l1_oBA?rlkey=6d063m5gy01un4at2mx0eocmx&st=ukdpih91&dl=0
)

Goal:

Dash these segments sequentially, as the next segment becomes fully written and ready.

Example  commands:
````
# First  copy seg_000[1,2,3...].mp4 to input mp4
# Then:
MP4Box  -dash 10000
		-dash-ctx ./stream/context.xml
		-profile dashavc264:live
		-mpd-refresh 20
		-dynamic
		-bs-switching no
		-segment-timeline
		-url-template		
		-time-shift -1
                input.mp4#video
		input.mp4#audio
		-out ./stream/stream.mpd
````
Estimated result: the dash manifest plays in dash.js player or[ reference dash-if player](https://reference.dashif.org/dash.js/latest/samples/dash-if-reference-player/index.html). When the next segment is adding to manifest, it is playing next.
Obtained result:  dash.js  plays only first 10s fragment, and then hangs.

I tried:
* changing mpd-refresh
* changing time-shift
* changing profile to live and back to dashavc264 live
* adding/replacing segment-timeline and url-template
 * adding :sreg for mpd
* adding -subdur
* adding -frag
* changing dash length
* adding min-buffer
* adding :ddur for sources
* adding ..everything i found
* removing `input.mp4#audio` source *---  WHOA! it's helped*!  But I don't hear sound )

Addendum: when I remove -dash-ctx, it plays with sound, but loops around the initial fragment, as expected. I have a vague feeling that there is a problem somewhere in keeping sound in dash-context. It is surely my musings and no more.

What helped else?

When I substitute to source the actual filename, instead of renaming it to "input.mp4", then,  everything works.
*But*, the `stream.mpd` and `context.xml` are constantly growing, because there are emerging new `Period` xml-entities. I've posted about it in the issue #2801 

So, it would be cool if you fix that problem with live streaming with sound.

Cheers!

**Comment by alex-rsk** [comment]
It would be great at least to have a comment about confirmation or disproval of that bug from @jeanlf .

**Comment by rbouqueau** [comment]
Thanks again for reporting and documenting the issue. We're slower than usual for processing issues these days but it doesn't mean we don't welcome them nor don't plan to do our best to address them. Thanks for your patience.

**Comment by jeanlf** [comment]
Indeed, the second stream was not correctly reloaded , now fixed thanks for the report (and check #2801)

**Comment by alex-rsk** [comment]
@jeanlf Thank you very much!
